### PR TITLE
feat: added basic cli app

### PR DESCRIPTION
**Pull request by irshadCypherock**

### CODE REVIEW

File: apps/cli/src/commands/device/update.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
+// eslint-disable-next-line import/order
```
Comment (ghost) [inline apps/cli/src/commands/device/update.ts:1]: Why is this required?

File: apps/cli/src/commands/wallet/sync.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+  static examples = [`$ <%= config.bin %> <%= command.id %>`];
+
+  async run(): Promise<void> {
+    const connection = await createConnection();
+    const app = await ManagerApp.create(connection);
+
+    const { walletList } = await app.getWallets();
+
+    this.logJson(walletList.map(e => ({ ...e, id: uint8ArrayToHex(e.id) })));
```
Comment (ghost) [inline apps/cli/src/commands/wallet/sync.ts:20]: May need to destroy connection after this

File: README.md [hunk 3]
```diff
@@ -80,13 +81,19 @@ pnpm start -s  # Start desktop application with short log format
...
 - `pnpm pretty`: Prettify all files
 - `pnpm build:submodules`: Build submodules if you are changing them
 
+- Desktop App
+    - `pnpm start`: Start desktop application (Hot reload enabled)
+    - `pnpm make`: Created packaged desktop application
+
+- CLI App
+    - `pnpm dev:cli -- <ARGS>`: Run cli application
+    - `pnpm make:cli`: Created packaged cli application
```
Comment (ghost) [inline README.md:95]: Need to add the requirement for pkg in the readme

**Review by ghost — CHANGES_REQUESTED** [review]

File: apps/cli/src/commands/device/update.ts (hunk 1 above)
Comment (irshadCypherock) [reply apps/cli/src/commands/device/update.ts:1]: If we don't add this, eslint will change the order of import as provided in the eslint config file. We don't want that to happen here. Since `config` should be imported before importing anything else to set the `process.env` variables.

**Review by ghost — DISMISSED** [review]

**Review by ghost — APPROVED** [review]
